### ISSUE TITLE
Hyphens in file names throws error

### ISSUE THREAD

**Issue by johanalkstal** [rollup/rollup-plugin-svelte#2]
If I use a hyphen in a file name - `my-component.html` - Svelte throws an error.

https://github.com/sveltejs/svelte/blob/master/src/validate/index.js#L47

I can compile it with `svelte-cli` so I'm thinking this has something to do with how this plugin compiles components?

```
Error: options.name must be a valid identifier
    at validate (D:\Projects\svelte-test\node_modules\svelte\src\validate\index.js:47:17)
    at Object.compile (D:\Projects\svelte-test\node_modules\svelte\src\index.js:39:20)
    at Object.transform (D:\Projects\svelte-test\node_modules\rollup-plugin-svelte\dist\rollup-plugin-svelte.cjs.js:24:18)
    at D:\Projects\svelte-test\node_modules\rollup\src\utils\transform.js:19:35
    at process._tickCallback (internal/process/next_tick.js:103:7)
    at Function.Module.runMain (module.js:577:11)
    at startup (node.js:159:18)
    at node.js:444:3
```

**Comment by Rich-Harris** [comment]
Yep, svelte-cli sanitizes names beforehand. This plugin will need to do the same. I'll just copy the relevant logic over...

**Comment by Rich-Harris** [comment]
Fixed in 1.3.1. Thanks
